sklearn/manifold/tests/test_spectral_embedding.py

```diff
@@ -1,227 +1,21 @@
-    se = SpectralEmbedding(n_components=1, affinity="<unknown>",
-                           random_state=np.random.RandomState(seed))
-    assert_raises(ValueError, se.fit, S)
-
-
-def test_connectivity(seed=36):
-    # Test that graph connectivity test works as expected
-    graph = np.array([[1, 0, 0, 0, 0],
-                      [0, 1, 1, 0, 0],
-                      [0, 1, 1, 1, 0],
-                      [0, 0, 1, 1, 1],
-                      [0, 0, 0, 1, 1]])
-    assert_equal(_graph_is_connected(graph), False)
-    assert_equal(_graph_is_connected(csr_matrix(graph)), False)
-    assert_equal(_graph_is_connected(csc_matrix(graph)), False)
-    graph = np.array([[1, 1, 0, 0, 0],
-                      [1, 1, 1, 0, 0],
-                      [0, 1, 1, 1, 0],
-                      [0, 0, 1, 1, 1],
-                      [0, 0, 0, 1, 1]])
-    assert_equal(_graph_is_connected(graph), True)
-    assert_equal(_graph_is_connected(csr_matrix(graph)), True)
-    assert_equal(_graph_is_connected(csc_matrix(graph)), True)
-
-
-def test_spectral_embedding_deterministic():
-    # Test that Spectral Embedding is deterministic
-    random_state = np.random.RandomState(36)
-    data = random_state.randn(10, 30)
-    sims = rbf_kernel(data)
-    embedding_1 = spectral_embedding(sims)
-    embedding_2 = spectral_embedding(sims)
-    assert_array_almost_equal(embedding_1, embedding_2)
-
-
-def test_spectral_embedding_unnormalized():
-    # Test that spectral_embedding is also processing unnormalized laplacian correctly
-    random_state = np.random.RandomState(36)
-    data = random_state.randn(10, 30)
-    sims = rbf_kernel(data)
-    n_components = 8
-    embedding_1 = spectral_embedding(sims,
-                                     norm_laplacian=False,
-                                     n_components=n_components,
-                                     drop_first=False)
-
-    # Verify using manual computation with dense eigh
-    laplacian, dd = graph_laplacian(sims, normed=False, return_diag=True)
-    _, diffusion_map = eigh(laplacian)
-    embedding_2 = diffusion_map.T[:n_components] * dd
-    embedding_2 = _deterministic_vector_sign_flip(embedding_2).T
+import numpy as np
 
-    assert_array_almost_equal(embedding_1, embedding_2)
+@ignore_warnings
+def test_spectral_embeding_import():
+    def import_spe_cls():
+        from sklearn.manifold import SpectralEmbedding
+        SpectralEmbedding()
+    def import_spe_fn():
+        from sklearn.manifold import spectral_embedding
+        random_state = np.random.RandomState(36)
+        data = random_state.randn(10, 30)
+        sims = rbf_kernel(data)
+        spectral_embedding(sims)
+
+    assert_warns_message(DeprecationWarning, "spectral_embedding is deprecated",
+                         import_spe_fn)
+    assert_warns_message(DeprecationWarning, "SpectralEmbedding is deprecated",
+                         import_spe_cls)
```

Comment:
Apologies for the lack of clarity... earlier I had hastily assumed that the `DeprecationWarning` was being raised upon import, which is why I suggested the use of those two functions...

Could you instead do...

```py
@ignore_warnings
def test_spectral_embeding_import():
        random_state = np.random.RandomState(36)
        data = random_state.randn(10, 30)
        sims = rbf_kernel(data)
        spectral_embedding(sims)

    assert_warns_message(DeprecationWarning, "spectral_embedding is deprecated",
                         spectral_embedding, sims)
    assert_warns_message(DeprecationWarning, "SpectralEmbedding is deprecated",
                         SpectralEmbedding)
```